### PR TITLE
Profile Settings: Add a Gravatar privacy section

### PR DESCRIPTION
**Pull request by ice9js**
#### Changes proposed in this Pull Request

These changes aim to improve user awareness of Gravatar being used for their profiles and to make its privacy settings more accessible by including them on the profile settings page on WordPress.com. See p8Slzc-6j-p2.

This change will require changes in WP.com's API and Gravatar itself to work but I'd like to agree on the functionality and wording before pushing those updates out.

The changes include a new **Profile Privacy** section which allows users to hide their profiles or disable their Gravatars altogether.  
The latter would make users' images and profile information inaccessible to third parties, but not delete it - that'd have to be done manually through Gravatar.com if they so desire.

There's also a link to the account settings on Gravatar.com at the bottom of the page.

calypso-me-settings

#### Testing instructions

TBA

**Note:** The form doesn't work yet as it requires changes on WP.com which may depend on what interface we end up with.

### CODE REVIEW

**Comment by ice9js** [comment]
Following the discussions with @hollyannehogan I moved the 'disable' function into its own card hoping it'd help make things more clear.

This is now ready for a copy and design review.

calypso-me

**Comment by michelleweber** [comment]
"WordPress.com uses Gravatar... This enables other sites and services to use your image" reads as though the reason other sites/servicss use your image is because of WP, not because of Gravatar, so I'd rephrase. Maybe something like:
- WordPress.com uses Gravatar, a universal avatar service, as your profile. When you sign up with Gravatar, other Gravatar-enabled sites will also be able to use this image and profile information.

For the next bit, tiny edits:
- Change the visibility of your profile using the options below, and visit Gravatar.com to access your full account settings. 

And for disabling, I'd love to change the order:
- Disabling Gravatar deletes your Gravatar data, and sites will no longer have access to your images and profile. 

Small fixes:
- "third-party sties" needs a dash
- Let's use "re-enable" and "re-enabling"

**Comment by ice9js** [comment]
Thanks for taking a look @michelleweber! Just one more question:

For the `WordPress uses Gravatar...` section: currently when you sign up for WordPress.com you automatically get an associated Gravatar account. It's one of the things we're trying to clear up.  
So maybe it'd be better not to use `When you sign up with Gravatar,...` because technically you're already signed up.

**Comment by michelleweber** [comment]
So maybe more like:

_WordPress.com uses Gravatar, a service that gives you a single profile to use across a range of sites, to manage your image and profile. Other Gravatar-enabled sites will also be able to use this image and information._

(This also gets rid of "a universal avatar service," which doesn't really mean anything to most people. Better to just use plain language.)
